[TW-1224] Update screenshots in Editing an API definition (#5017)
postmanlabs/postman-docs: src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md

If your API doesn't have a definition, you can generate an example definition that you can edit.

1. Select **APIs** in the sidebar and select an API.

> 1. From the API's menu, select **Add definition > Author from scratch**. Alternatively, select **create** in the sidebar under the API.

Reviewer: ```suggestion
1. From the API's overview, next to **Definition**, select **+** and select **Author from scratch**. Alternatively, select **create** in the sidebar under the API.
```

> 1. Select the definition type and format. We are using OpenAPI and YAML in this example.

Reviewer: Do we want to mention "We are using OpenAPI and YAML in this example."? My concern is that this sentence is dependent on the screenshot loading for the user. We could maybe say "The default selections are ...", but we might run the risk of the UI changing, and then the docs becoming a little out of date.

> 1. Select the definition type and format. We are using OpenAPI and YAML in this example.

Reviewer 2: ```suggestion
1. Select the definition type and format. This example uses OpenAPI and YAML.
```

Reviewer 2: Style guide says to avoid using we and our in the core docs (as opposed to tutorials and blog posts).

1. To start with a sample definition, select **Use a boilerplate**.

>     <img alt="Generating an API definition" src="https://assets.postman.com/postman-docs/v10/api-builder-author-definition-v10.17.jpg" width="323px"/>

Reviewer: What do you think about making the screenshot a little bigger? I'm suggesting this because the text in the screenshot is a little difficult to read. I tested the image at `400` and I think it's easier to read.

>     <img alt="Generating an API definition" src="https://assets.postman.com/postman-docs/v10/api-builder-author-definition-v10.17.jpg" width="400px"/>

Reviewer 3: The screenshot link is broken for me. I checked the S3 bucket and don't see `api-builder-author-definition-v10.17.jpg` in it

Author: Uploaded and fixed the border!
